src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -1,23 +1,41 @@
 package frc.robot.subsystems.elevator;
 
+import edu.wpi.first.wpilibj2.command.Command;
+
 public class ElevatorConstants {
-    public static final double kP = 0; // TODO
-    public static final double kI = 0; // TODO
-    public static final double kD = 0; // TODO
-    public static final double tolerance = 0; // TODO
+    public static final double KP = 42;
```

Comment:
Looks like your convention for constant (final) variables is uppercase snake_case (also known as SCREAMING_SNAKE_CASE)
I've noticed some places in the code that do not follow those conventions, if that matters to you